packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -16,3 +19,25 @@ export type TPasswordResetAndTradingPasswordManager = {
     platform: 'mt5' | 'dxtrade';
     account_group: 'real' | 'demo';
 };
+
+export type TResetPasswordIntent = {
+    current_list: Array<DetailsOfEachMT5Loginid> & { [key: string]: DetailsOfEachMT5Loginid };
```

Comment:
```suggestion
    current_list: Array<DetailsOfEachMT5Loginid> & Record<string, DetailsOfEachMT5Loginid>;
```
https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type
Consider using the Record utility type to improve readability.